### PR TITLE
http: moving exceptions out of http_server_properties_cache_manager_impl

### PR DESCRIPTION
**Pull request by alyssawilk**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
part of https://github.com/envoyproxy/envoy-mobile/issues/176

### CODE REVIEW

File: source/extensions/filters/http/alternate_protocols_cache/filter.cc [hunk 1]
```diff
@@ -38,6 +38,7 @@ Filter::Filter(const FilterConfigSharedPtr& config, Event::Dispatcher& dispatche
     : cache_(config->getAlternateProtocolCache(dispatcher)), time_source_(config->timeSource()) {}
 
 Http::FilterHeadersStatus Filter::encodeHeaders(Http::ResponseHeaderMap& headers, bool) {
+
```
Comment (RyanTheOptimist) [inline source/extensions/filters/http/alternate_protocols_cache/filter.cc:41]: nit: Remove this?

File: tools/code_format/config.yaml [hunk 2]
```diff
@@ -131,7 +131,6 @@ paths:
     - source/common/http/filter_chain_helper.h
     - source/common/http/conn_manager_utility.cc
     - source/common/http/match_delegate/config.cc
-    - source/common/http/http_server_properties_cache_manager_impl.cc
```
Comment (RyanTheOptimist) [inline tools/code_format/config.yaml:134]: Oh, rad!

**Review by RyanTheOptimist — APPROVED** [review]
Most excellent!

File: source/extensions/filters/http/alternate_protocols_cache/filter.cc (hunk 1 above)
Comment (alyssawilk) [reply source/extensions/filters/http/alternate_protocols_cache/filter.cc:41]: Given I've got another PR in this space and github is broken I'll do in my follow up :-)

**Pull request merged**